### PR TITLE
Re-structure pipeline

### PR DESCRIPTION
**Pull request by KateSakharova**
Restructured pipeline to nf-core view
- created main.nf main entry
- added virify.nf to workflows and simplified
- all subworkflows wrapped into separate files
- modules moved to local/name/main.nf
- publishDir moved to modules.config
- container specified in each module main.nf 
- moved help to separate module
- moved warning to separate module
- removed CWL (woohoo)
- added samplesheet support
- removed --list option (replaced with samplesheet)
- replaced name with meta (resolved issues with --list option)
- added some nf-core tools

Now virify supports samplesheet (fq1 and fq2 are optional) 
id,assembly,fq1,fq2

For assembly as input:
If you have assembly as input you can use old argument --fasta - that will run pipieline for 1 assembly.
If you have more than 1 assembly - use samplesheet.

For raw_reads as input:
You need to use ONLY samplesheet (even for 1 run). I replaced --illumina option with --assemble (boolean). User should specify that flag to force assembly process (that flag will help user to understand what he is doing). _Hopefully, nobody uses it because it is experimental feature._

### CODE REVIEW

File: subworkflows/local/annotate.nf [hunk 1]
```diff
@@ -93,25 +93,36 @@ workflow ANNOTATE {
...
     CHECKV(
-      predicted_contigs.combine( contigs.map { name, fasta -> fasta }),
+      predicted_contigs.join(contigs.map { meta, fasta -> fasta }),
       checkv_db
     )
 
-    viphos_annotations = ANNOTATION.out.map { _, __, annotations -> annotations }.collect()
-    taxonomy_annotations = ASSIGN.out.map { _, __, taxonomy -> taxonomy }.collect()
-    checkv_results = CHECKV.out.map { _, __, quality_summary, ___ -> quality_summary }.collect()
+    viphos_annotations = ANNOTATION.out.annotations.groupTuple().map{
```
Comment (mberacochea) [inline subworkflows/local/annotate.nf:100]: Is this `groupTuple` required becase `ANNOTATION` return a list of files? I'm not sure I follow

**Comment by mberacochea** [comment]
So far.. it look goods to me :) Thank you @KateSakharova

File: subworkflows/local/annotate.nf (hunk 1 above)
Comment (KateSakharova) [reply subworkflows/local/annotate.nf:100]: it is in development stage yet, I'm not sure what it returns either :D

Comment (mberacochea) [reply subworkflows/local/annotate.nf:100]: Perfect hehe

**Pull request merged**